omDomainJSON.py

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright 2023 The Superpower Institute Ltd.
+#
+# This file is part of OpenMethane.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Generate a JSON file describing the domain and grid.
+"""
+
+import numpy as np
+import netCDF4 as nc
+from omInputs import domainXr, domainProj
+from omOutputs import domainJSONOutputPath
+import json
+
+def makePoint(x, y):
+    return ([float(x), float(y)])
+
+def writeDomainJSON():
+    # Load raster land-use data
+    print("converting domain grid details to JSON")
+
+    domain = {
+        "crs": {
+            "projection_type": "lambert_conformal_conic",
+            "standard_parallel": float(domainXr.attrs['TRUELAT1']),
```

Comment:
I've used property names from CF Conventions here, instead of the attribute names from the domain NetCDF file, which is a subjective decision.

I found the attribute naming in the domain file very hard to understand and required reading Python scripts that used the file to find their meaning. My hope is that these standardised, more descriptive names will be more useful.

Happy to be discuss if this seems like moving in the wrong direction.

Reply:
I'm not suggesting changing the working NetCDF files. I was more suggesting it might be confusing to use completely different names for the same data in different files the way I've done here.

I'm not sure if this file counts as "public facing" since I don't imagine anyone else would need a JSON file of the details that are already available in NetCDF.